submissions/OpemipoBanso/CSS/HTML and CSS tasks/html/kasuwa homepage.html

```diff
@@ -0,0 +1,526 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="C:\Users\USER\Documents\Kasuwa_files\css\kasuwa-homepage.css">
```

Comment:
Why are your css and image files in a totally different directory? @Verdantcoderr

Reply:
Yeah @Verdantcoderr, I think it will be better to put the css files in a css folder in the directory where the html files so you can reference them relatively.